redpanda/values.yaml

```diff
@@ -13,135 +13,201 @@
+  schemaRegistry:
+    enabled: true
+    all:
+    - address: 0.0.0.0
       name: schema-registry
       port: 8081
       external:
-        enabled: true
-
-  rpk:
-    # Enables memory locking.
-    enable_memory_locking: false
-
-    # Send usage stats back to vectorized
-    enable_usage_stats: true
-
-    # This should be set to true unless with have CPU affinity
-    # https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/#static-policy
-    #
-    # Equivalent to passing the following to redpanda
-    # --idle-poll-time-us 0 --thread-affinity 0 --poll-aio 0
-    overprovisioned: true
-
-    # Increases the number of allowed asynchronous IO events
-    tune_aio_events: false
-
-    # Syncs NTP
-    tune_clocksource: false
-
-    # Installs a custom script to process coredumps and save them to the given directory.
-    tune_coredump: false
-
-    # Disables hyper-threading, sets the ACPI-cpufreq governor to 'performance'. Additionaly
-    # if system reboot is allowed: disables Intel P-States, disables Intel C-States,
-    # disables Turbo Boost
-    #
-    # This is not possible to set to true while running in a container.
-    tune_cpu: false
-
-    # Distributes IRQs across cores with the method deemed the most appropriate for the
-    # current device type (i.e. NVMe)
-    #
-    # This is not possible to set to true while running in a container.
-    tune_disk_irq: false
+        enabled: false
+      tls:
+        # enabled: true
+        cert: cert1
+        requireClientAuth: false
+#
+# Log level
+# Valid values are info, warn, debug, trace
+logLevel: info
+#
+# Persistence
+# TODO modify to conform to https://github.com/bitnami/charts/blob/master/bitnami/kafka/values.yaml#L888
+storage:
+  # TODO make use of enabled parameter
+  enabled: true
+  # Absolute path on host to store Redpanda's data.
+  # If not specified, then `emptyDir` will be used instead.
+  # If specified, but `persistentVolume.enabled` is `true`, then has no effect.
+  hostPath: ""
+  # If `enabled` is `true` then a PersistentVolumeClaim will be created and
+  # used to store Redpanda's data, otherwise `hostPath` is used.
+  persistentVolume:
+    # TODO possibly replace with storage.enabled
+    enabled: true
+    # TODO lower default size to a more reasonable size (10 - 50 Mi)
+    size: 100Gi
+    # If defined, then `storageClassName: <storageClass>`.
+    # If set to "-", then `storageClassName: ""`, which disables dynamic
+    # provisioning.
+    # If undefined or empty (default), then no `storageClassName` spec is set,
+    # so the default provisioner will be chosen (gp2 on AWS, standard on
+    # GKE, AWS & OpenStack).
+    storageClass: ""
+    # Additional labels to apply to the created PersistentVolumeClaims.
+    labels: {}
+    # Additional annotations to apply to the created PersistentVolumeClaims.
+    annotations: {}
 
-  seastar:
-    default_log_level: info
+# Redpanda Keeper configuration
+rpk:
```

Comment:
Perhaps we add a host tuner daemonset that can be disabled by default but enabled by the user.